libnetwork/network.go

```diff
@@ -1455,6 +1455,10 @@ func (n *Network) getSvcRecords(ep *Endpoint) []etchosts.Record {
 		return nil
 	}
 
+	// TODO(robmry) - should this also look at sr.svcIP6Map?
```

Comment:
As you noted in your comment, IPv6 addresses aren't removed explicitly but they're still removed. I think it's okay to keep this behavior but your comment is much needed to clarify what's going on. Can you remove this TODO and keep the rest of the comment please?

Reply:
Sure - done.